v3/src/data-interactive/handlers/collection-handler.ts

```diff
@@ -4,8 +4,8 @@ import { getSharedCaseMetadataFromDataset } from "../../models/shared/shared-dat
 import { toV2Id } from "../../utilities/codap-utils"
 import { registerDIHandler } from "../data-interactive-handler"
 import {
-  DIHandler, DIResources, DIValues, DICreateCollection, DICollection, DIUpdateCollection
-} from "../data-interactive-types"
+  DIHandler, DIResources, DIValues} from "../data-interactive-types"
```

Comment:
Should fit on one line now.

Reply:
I'll add that fix to the next PR: https://github.com/concord-consortium/codap/pull/1755